src/cryptonote_config.h

```diff
@@ -182,8 +182,10 @@
 #define HF_VERSION_EXACT_COINBASE               13
 #define HF_VERSION_CLSAG                        13
 #define HF_VERSION_DETERMINISTIC_UNLOCK_TIME    13
+#define HF_VERSION_2021_SCALING                 17
 
 #define PER_KB_FEE_QUANTIZATION_DECIMALS        8
+#define CRYPTONOTE_SCALING_2021_FEE_ROUNDING_PLACES 3
```

Comment:
Originally I recommended 2 significant digits. Has there been any discussion around this, reasoning for 3 vs 2?